build.gradle

```diff
@@ -18,10 +18,11 @@ version plugin_version
 repositories {
     mavenCentral()
     jcenter()
-    maven { url 'https://hub.spigotmc.org/nexus/content/groups/public/' }
```

Comment:
Was it really the spigot repo causing issues or does this still work if we keep it uncommented?

Reply:
They fixed that repo, so it works again when uncommented

Reply:
If this repo is needed, leave it uncommented. If it is not we can remove it

Reply:
Alright leave it uncommented but keep the other one I told you to add just in case. It will fallback on it if spigot ever does a 🐒 again